academy/4-my-own-chain/create-handling.md

```diff
@@ -1,11 +1,11 @@
 ---
-title: Create the Game Handling
+title: Creating and Saving a Game Properly
```

Comment:
@rob-Hitchens Mmh, I was thinking of "proper" as the action itself, because the section before is only about creating the message `MsgCreateGame, which in and of itself does not create nor save a game.
If you have a better idea about this distinction.